workers/unity/Packages/io.improbable.gdk.core/Subscriptions/WriterSubscriptionManager.cs

```diff
@@ -9,17 +9,19 @@ namespace Improbable.Gdk.Subscriptions
     public abstract class WriterSubscriptionManager<TComponent, TWriter> : SubscriptionManager<TWriter>
         where TWriter : IRequireable where TComponent : ISpatialComponentData
     {
-        private readonly ComponentUpdateSystem componentUpdateSystem;
+        private readonly EntityManager entityManager;
         private Dictionary<EntityId, HashSet<Subscription<TWriter>>> entityIdToWriterSubscriptions;
 
         private readonly HashSet<EntityId> entitiesMatchingRequirements = new HashSet<EntityId>();
         private readonly HashSet<EntityId> entitiesNotMatchingRequirements = new HashSet<EntityId>();
 
         private static readonly uint ComponentId = ComponentDatabase.GetComponentId<TComponent>();
+        private static readonly ComponentType ComponentType = ComponentDatabase.Metaclasses[ComponentId].Data;
+        private static readonly ComponentType ComponentAuthType = ComponentDatabase.Metaclasses[ComponentId].Authority;
```

Comment:
Use `ComponentDatabase.GetMetaclass(componentId)` instead? It gives a more helpful error if this ever was to throw an error